### PR TITLE
[UI] Changed priority of fonts

### PR DESCRIPTION
**Pull request by AButler**
Use Arial by default as the font for Ace

### CODE REVIEW

**Comment by kasperp** [comment]
@AButler why remove Helvetica and resort the default sans-serif font on e.g. ipads or on a mac ?

**Comment by AButler** [comment]
> why remove Helvetica and resort the default sans-serif font on e.g. ipads or on a mac ?

It was a request from @russelldocking since Ace didn't render nicely using the Helvetica fonts. WIll check with him next week to find the reason.

**Comment by russelldocking** [comment]
@kasperp my thought was that for the few users that might have Helvetica on their PC, the experience is not as good as it would be in Arial, and we would expect far fewer visitors in an engineering environment to be using a mac anyway. Macs generally have Arial too, so that should not be a problem. 

If we're bothered about allowing those users who are on macs to see everything in Helvetica by default, we could swap the font stack around so Arial is first, i.e. font-family: Arial, Helvetica, sans-serif; - the configit.com website does this already.

But if "we" have already decided that Arial is what we want to use for everything (as per the Configit identity guidelines) then we should probably stick to our guns :)

Having said all that, I'd love it if we could use a new font for the body text on Ace, Quote and the Configit website - one that has more weights and thus gives us more flexibility. One for Core UI v2.0 I think!

**Comment by russelldocking** [comment]
Sounds good to me - @AButler what do you reckon?

**Comment by AButler** [comment]
Yeah, makes sense - will change

**Comment by AButler** [comment]
@russelldocking @kasperp Have made the change

File: less/variables.less [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 // Typography
 // -------------------------
-@sansFontFamily:        "Helvetica Neue", Helvetica, Arial, sans-serif;
+@sansFontFamily:        Arial, "Helvetica Neue", Helvetica, sans-serif;
```
Comment (kasperp) [inline less/variables.less]: Sorry -  I don't think "Helvetica Neue" shouldn't be in the stack

Comment (chad-configit) [inline less/variables.less]:
```javascript
!!( 'I don't think "Helvetica Neue" shouldn't be in the stack' )
```
:smile_cat:

Comment (kasperp) [inline less/variables.less]: blush

**Pull request merged**